compressor/filters/base.py

```diff
@@ -40,6 +40,13 @@ class FilterBase(object):
     Subclasses should implement `input` and/or `output` methods which must
     return a string (unicode under python 2) or raise a NotImplementedError.
     """
+
+    # Since precompiling moves files around, it breaks url()
+    # statements in css files. therefore we run the absolute and relative filter
+    # on precompiled css files even if compression is disabled.
+    # This flag allows those filters to do so.
+    run_with_compression_disabled = False
```

Comment:
i would keep the comment only here and remove it everywhere else.

Reply:
Agreed